src/common/com/intellij/plugins/haxe/util/HaxeResolveUtil.java

```diff
@@ -526,8 +526,13 @@ public static HaxeClass tryResolveClassByQName(@Nullable PsiElement type) {
       return null;
     }
 
+    // can be accessed without full QName
+    HaxeClass result = findClassByQName(type.getText(), type.getContext());
```

Comment:
@eliasku I seen it,

Actually interesting info about completion, we might need a test for that too. Yeah good idea.